src/components/ScreenWrapper/propTypes.js

```diff
@@ -59,6 +65,8 @@ const defaultProps = {
     shouldEnablePickerAvoiding: true,
     shouldShowOfflineIndicator: true,
     offlineIndicatorStyle: [],
+    shouldDisableFocusTrap: false,
+    shouldDisableAutoFocus: false,
```

Comment:
Hmm, I don't think so.

Those props have slightly different behavior:
- `shouldDisableFocusTrap` - disables the `FocusTrap` completely,
- `shouldDisableAutoFocus` - disables only the autofocus option of the `FocusTrap` (the trap is still active).

Let me give some examples, of how it is used:
- the `SearchPage` has to have the `FocusTrap` **enabled** (the user shouldn't be able to escape from the RHP), but it has to have the autofocus of focus trap **disabled** (the autofocus of the input bar has higher priority).
- i.g. the `ReportScreen` has to have the `FocusTrap` disabled because the user should be able to _tab_ to the list of the reports (`BaseSidebarScreen`)  (in this case, the autofocus prop doesn't matter).

Reply:
The `FocusTrap` will not focus on the custom `<View>` component. Instead, other logic will handle focusing. In the case of the `SearchPage` it will be the input bar.